integration-tests/src/test/java/com/datastax/oss/driver/api/core/tracker/RequestLoggerIT.java

```diff
@@ -139,6 +137,13 @@
                   .build())
           .build();
 
+  @Rule
+  public TestRule chain =
+      RuleChain.outerRule(simulacronRule)
+          .around(sessionRuleRequest)
+          .around(sessionRuleNode)
+          .around(sessionRuleDefaults);
```

Comment:
Strictly speaking the 3 session rules don't require any ordering with respect to each other, but that's the only way to embed them all into the simulacron rule.